docs/xml.md

```diff
@@ -561,6 +562,65 @@ A `Ref` value pointing to an `Item` with a `referent` of `RBX466F72207262782D646
 ```
 
+### SecurityCapabilities
+
+The `SecurityCapabilities` data type represents a set of permissions or capabilities, encoded as a bit field, that dictate whether a script can perform specific actions within the Roblox environment. Each bit in the field corresponds to a particular capability.
+
+Roblox uses this type to manage script capabilities for certain instances like `Folder`, `Model`, and `Script`. Although the property is present on all instances, its functionality is only exposed for specific classes. The encoding of `SecurityCapabilities` as a bit field aligns with the permissions previously managed by identity permissions in the lowest 8 bits.
+
+A `SecurityCapabilities` value with no capabilities set appears as follows:
+
+```xml
+<SecurityCapabilities name="Example">0</SecurityCapabilities>
+```
+
+A `SecurityCapabilities` value with `RunClientScript` (2 ^ 8 = 256) & `RunServerScript` (2 ^ 9 = 512) capabilities set appears as follows:
+
+```xml
+<SecurityCapabilities name="Example">768</SecurityCapabilities>
+```
+
+The following table maps each bit in the field to a corresponding capability:
+
+| **Bit Position** | **Capability**       | **Description**                                                                                     |
+|------------------|----------------------|-----------------------------------------------------------------------------------------------------|
+| 0                | Plugin               | Provides access to instances related to plugins.                                                    |
+| 1                | LocalUser            | Members with this security are usually intended for internal usage, but developers are allowed to access it through the command bar. |
+| 2                | WritePlayer          | Enables writing to `Player` class instances.                                                        |
+| 3                | RobloxScript         | Indicates a Roblox internal script.                                                                 |
+| 4                | RobloxEngine         | Accesses engine-level features.                                                                     |
+| 5                | NotAccessible        | Members with this security are intended to only be written to outside of a running game.            |
+| 8                | RunClientScript      | [Allows execution of LocalScript or Script with a RunContext value of Client on the client.](https://create.roblox.com/docs/scripting/capabilities#execution-control) |
+| 9                | RunServerScript      | [Allows execution of Script with a RunContext value of Server on the server.](https://create.roblox.com/docs/scripting/capabilities#execution-control) |
+| 11               | AccessOutsideWrite   | [Allows fetching and receiving instances from outside the sandboxed container.](https://create.roblox.com/docs/scripting/capabilities#instance-access-control) |
```

Comment:
I would prefer that these links be split out of this table and placed at the bottom, as it makes aligning the table + reading it easier.